ARK Server Manager/PlayerProfile.xaml.cs

```diff
@@ -0,0 +1,158 @@
+    /// <summary>
+    /// Interaction logic for PlayerProfile.xaml
+    /// </summary>
+    public partial class PlayerProfile : Window
+    {
+        public PlayerProfile(PlayerInfo player, String serverFolder)
+        {
+            InitializeComponent();
+
+            this.Player = player;
+            this.ServerFolder = serverFolder;
+            this.DataContext = this;
+        }
+
+        public PlayerInfo Player
+        {
+            get;
+            private set;
+        }
+
+        public Player ArkDataPlayer
+        {
+            get
+            {
+                return Player?.ArkData;
+            }
+        }
+
+        public Tribe ArkDataTribe
+        {
+            get
+            {
+                return Player?.ArkData?.Tribe;
+            }
+        }
+
+        public String CreatedDate
+        {
+            get
+            {
+                return ArkDataPlayer?.FileCreated.ToString("G");
+            }
+        }
+
+        public Boolean IsTribeOwner
+        {
+            get
+            {
+                return ArkDataPlayer != null && ArkDataTribe != null && ArkDataTribe.OwnerId == ArkDataPlayer.Id;
+            }
+        }
+
+        public String PlayerLink
+        {
+            get
+            {
+                if (String.IsNullOrWhiteSpace(ServerFolder))
+                    return null;
+
+                return String.Format("/select, {0}", Path.Combine(ServerFolder, Config.Default.SavedArksRelativePath, String.Format("{0}.arkprofile", Player.SteamId.ToString())));
+            }
+        }
+
+        public String ProfileUrl
+        {
+            get
+            {
+                return ArkDataPlayer?.ProfileUrl;
+            }
+        }
+
+        public String ServerFolder
+        {
+            get;
+            private set;
+        }
+
+        public String TribeLink
+        {
+            get
+            {
+                if (String.IsNullOrWhiteSpace(ServerFolder))
+                    return null;
+                
+                return ArkDataTribe == null ? null : String.Format("/select, {0}", Path.Combine(ServerFolder, Config.Default.SavedArksRelativePath, String.Format("{0}.arktribe", ArkDataTribe.Id.ToString())));
+            }
+        }
+
+        public String TribeOwner
+        {
+            get
+            {
+                return ArkDataTribe != null && ArkDataTribe.Owner != null ? string.Format("{0} ({1})", ArkDataTribe.Owner.SteamName, ArkDataTribe.Owner.CharacterName) : null;
+            }
+        }
+
+        public String UpdatedDate
+        {
+            get
+            {
+                return ArkDataPlayer?.FileUpdated.ToString("G");
+            }
+        }
+
+        public String WindowTitle
+        {
+            get
+            {
+                return String.Format("Player Profile - {0}", Player.SteamName);
+            }
+        }
+
+        private void SteamName_RequestNavigate(object sender, RequestNavigateEventArgs e)
```

Comment:
These three blocks of code look suspiciously similar.  Does RequestNavigateEventArgs not contain the link to navigate to?

Also, does Hyperlink support Command?  If so, you can pass the link as the command arg that way.